### ISSUE TITLE
Disable plugin for project

### ISSUE THREAD

**Issue by benoitverstraete** [long1eu/flutter_i18n#67]
Hi,

Is it possible to disable the plugin for given project ? I would like to enable the plugin only for particular projects, not for all projects in android studio.

For example, I created a flutter package and I don't want the class S to be generated ..

Thanks a lot !

**Comment by noordawod** [comment]
This is a good suggestion indeed, thanks.
Do you want to submit a pr for this, @benoitverstraete?

**Comment by benoitverstraete** [comment]
Thanks @noordawod !
I didn't work on a solution so I can't submit a new PR.

**Comment by noordawod** [comment]
This will be fixed once #98 is merged.

**Comment by noordawod** [comment]
This is now resolved in v1.1.0. Please upgrade through the Plugins page in your IDE and read the new README.
